### PR TITLE
[DC-2024] Initialize the Washington DC 2024 event

### PR DESCRIPTION
**Pull request by gallimorej**
Get everything set up for the Washington DC event.

Used the `add-new-event.sh` script to get it started.

Copied info from 2023 event as appropriate.

### CODE REVIEW

**Comment by gallimorej** [comment]
Also... I sent an email to info-at-devopsdays.org with all the info for updates to the email list and the Slack channel.

File: data/events/2024-washington-dc.yml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+ # - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (phrawzty) [inline data/events/2024-washington-dc.yml:53]: You have 12 people in your mailing list (as per the email you sent), but only ten listed here. Please either add the other two people to this section, or remove them from the mailing list.

**Review by phrawzty — CHANGES_REQUESTED** [review]
Looks great! Just one item to address. 🙂

File: data/events/2024-washington-dc.yml (hunk 1 above)
Comment (gallimorej) [reply data/events/2024-washington-dc.yml:53]: Thanks for the catch there. I updated the YAML file (should have 11 team members) and resent the email (with 11 team members) so they should match now.

**Comment by gallimorej** [comment]
@phrawzty I think I reconciled everything (YAML and email). Ready for a re-review when you have a moment.

**Review by phrawzty — APPROVED** [review]

**Pull request merged**